canonical/lxd-ui
# feat(logo) use microcloud logo, in case user.microcloud server setting is present

## Done

- use microcloud logo, in case user.microcloud server setting is present

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @mas-who or @edlerd for access.
    - With a local copy of this branch, [build and run as described in the docs](../CONTRIBUTING.md#setting-up-for-development).
2. Perform the following QA steps:
    - set the key `lxc config set user.microcloud=true` and check the ui is switching to the microcloud logo
    - remove the key again with `lxc config unset user.microcloud`

## Screenshots

![image](https://github.com/user-attachments/assets/a8f3fd13-1c2a-414a-83cf-6ce9ea42cc18)
